behind-the-veil-siteroot/imports/ui/components/DatabaseHelper.jsx

```diff
@@ -499,3 +501,33 @@ export function useUserBookings(username) {
   });
   return { isLoadingUserBooking, artistBookingData };
 }
+
+
+export function reviewCollection(username) {
+    // collect bookings data
+    const isLoadingBookings = useSubscribe("all_user_bookings", username);
+    const bookingsData = useTracker(() => {      
+        return BookingCollection.find({}).fetch();
+    });
+
+    // collect reviews data
+    const isLoadingReviews = useSubscribe("all_reviews",[]);
+    const reviewData = useTracker(() => {
+        return ReviewCollection.find({}).fetch();
+    })
+
+    // get the reviews relative to a specific artist
+    const reviewSourceArray = []
+    for (let i=0; i < reviewData.length; i++) {
+        for (let j = 0; j < bookingsData.length; j++) {
+            if (bookingsData[j]._id === reviewData[i].bookingId) {
+                reviewSourceArray.push(reviewData[i]);
+            }
+        }
+    }
```

Comment:
Instead of doing this, would it be easier to just get all the reviews ever given to this user by just using the username as a variable? like in all_user_bookings?